kpi/utils/mongo_helper.py

```diff
@@ -18,8 +18,23 @@ class MongoHelper:
     and KoBoCAT's ParseInstance class to query mongo.
     """
 
-    KEY_WHITELIST = ['$or', '$and', '$exists', '$in', '$gt', '$gte',
-                     '$lt', '$lte', '$regex', '$options', '$all']
+    OR_OPERATOR = '$or'
+    AND_OPERATOR = '$and'
+    IN_OPERATOR = '$in'
+
+    KEY_WHITELIST = [
+        OR_OPERATOR,
+        AND_OPERATOR,
+        IN_OPERATOR,
+        '$exists',
+        '$gt',
+        '$gte',
+        '$lt',
+        '$lte',
+        '$regex',
+        '$options',
+        '$all',
```

Comment:
:eyes:

Reply:
`$options` is a modifier for `$regex` and `$all` does [this](https://docs.mongodb.com/manual/reference/operator/query/all/):

The [`$all`](https://docs.mongodb.com/manual/reference/operator/query/all/#mongodb-query-op.-all) is equivalent to an [`$and`](https://docs.mongodb.com/manual/reference/operator/query/and/#mongodb-query-op.-and) operation of the specified values; i.e. the following statement:

```
{ tags: { $all: [ "ssl" , "security" ] } }
```

is equivalent to:

```
{ $and: [ { tags: "ssl" }, { tags: "security" } ] }
```